internal/cephfs/fsjournal.go

```diff
@@ -69,18 +82,28 @@ func checkVolExists(ctx context.Context, volOptions *volumeOptions, secret map[s
 	}
 	imageUUID := imageData.ImageUUID
 	vid.FsSubvolName = imageData.ImageAttributes.ImageName
+	var evnf ErrVolumeNotFound
+
+	clone, cloneInfoErr := getcloneInfo(ctx, volOptions, cr, volumeID(vid.FsSubvolName))
```

Comment:
What if volume is not a clone? This should be done iff pid or sid is non NULL. Also, will serve to understand if clone is complete